From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jian De <jiande2020@protonmail.com>
Date: Wed, 23 Sep 2026 00:00:00 +0800
Subject: [PATCH] example: support Dart 3.10

The AGL Flutter SDK currently ships Dart 3.10. The example does not use
APIs that require Dart 3.11, so lower its minimum SDK constraint.

Upstream-Status: Inappropriate [AGL toolchain compatibility]
Signed-off-by: Jian De <jiande2020@protonmail.com>
---
 example/flutter_phone_message/pubspec.yaml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/example/flutter_phone_message/pubspec.yaml b/example/flutter_phone_message/pubspec.yaml
index 95f67b9..7394814 100644
--- a/example/flutter_phone_message/pubspec.yaml
+++ b/example/flutter_phone_message/pubspec.yaml
@@ -19,7 +19,7 @@ publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 version: 1.0.0+1
 
 environment:
-  sdk: ^3.11.5
+  sdk: ^3.10.0
 
 # Dependencies specify other packages that your package needs in order to work.
 # To automatically upgrade your package dependencies to the latest versions
